styleguide/components/21-atoms/icons/icon/README.md

```diff
@@ -1,4 +1,12 @@
 # Gent icon font
 
+## General
+
 For more information on the use of the Gent icon font see the notes in the
 `Base -> Fonts section`
+
+## Web accessibility
+
+* Icons are not used as the only means to convey the information. They can only
+  be used when their context also defines their meaning.
```

Comment:
I believe this last sentence is a bit redundant.
I propose:
* Icons are not used as the only means to convey the information, i.e. icons are decorative.
* The icon must have the aria-hidden=true attribute.